assets/src/components/ui/entity-paper-frame/entity-paper-frame.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External imports
+ */
+import { withInstanceId } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import './style.css';
+import EntityCuid from './entity-cuid';
+
+/**
+ * EntityPaperFrame
+ * adds a styled frame that gives the appearance
+ * of a piece of paper on a surface
+ *
+ * @return {Object} Entity with added EntityPaperFrame
+ */
+const EntityPaperFrame = ( { instanceId, children, entityID } ) => {
+	return (
+		<div id={ `ee-entity-paper-frame-${ instanceId }` }
```

Comment:
`id` could have been in next line.

Reply:
oops... ya think I'll make that change before merging. thnx 👍

Reply:
We have used Prettier, ESLint and Husky in the last project and it was a huge time-saver in the context of formatting.